polygeist/tools/cgeist/Test/Verification/builtin.cpp

```diff
@@ -0,0 +1,13 @@
+// RUN: cgeist -O0 %s -w -o - -S --function=* | FileCheck %s
```

Comment:
Do we need the `function` filter at all, if we allow all functions?

Reply:
not sure, reuse an existing polygiest test

Reply:
Yep, it's needed unfortunately